controllers/ProjectController.php

```diff
@@ -172,11 +168,17 @@ public function actionCreate() {
             
         } else { //Sinon c'est qu'il faut afficher ce formulaire
             //Récupération de la liste des projets existants pour la dropdownList
+            //SILEX:conception
+            // This quick fix is used to show all users available in the 
+            // dropdown list but we need to implements an autocompletion
```

Comment:
Please don't add solution "but we need to implements an autocompletion" because we need to discuss.